File: WebApp/Models/PassengerType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class PassengerType
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }

        public double Discount { get; set; }

        public List<ApplicationUser> ApplicationUsers { get; set; }


    }
}

File: Projekat/src/app/passenger-type.ts
export class PassengerType {
    Id: number;
    Name: string;
    Discount: number;

    constructor(private id: number, private name: string, private discount:number) { 
        this.Id = id;
        this.Name = name;
        this.Discount = discount;
        
      }
}
